server/server.go

```diff
@@ -1,154 +0,0 @@
-package main
-
-import (
-	"fmt"
-	"io/ioutil"
-	"log"
-	"strconv"
-	"net/http"
-	"encoding/json"
-	"database/sql"
-
-	"github.com/willf/bloom"
-	"github.com/blics18/SendGrid/client"
-	_ "github.com/go-sql-driver/mysql"
-	
-)
-
-type bloomFilter struct {
-	filter *bloom.BloomFilter
-}
-
-func createBloomFilter() *bloomFilter {

```

Comment:
typically this would be an exported function and instead of create it is idiomatic to use new
```

type BloomFilter struct {
   filter *bloom.BloomFilter      // this still is "private" or not exported
}

func NewBloomFilter() *BloomFilter{
...
}

```